### PR TITLE
Only display old webhooks on the Incoming webhook logs page

### PR DESCRIPTION
**Pull request by pjlast**
The old Incoming webhook logs page displayed logs from the new webhooks, causing a lot of noise since they have no External Service attached. See screenshots

Before:

Screenshot 2022-12-08 at 16 00 55

After:

Screenshot 2022-12-08 at 16 05 49

## Test plan

Verified manually

### CODE REVIEW

File: client/web/src/site-admin/webhooks/backend.ts [hunk 1]
```diff
@@ -81,9 +81,18 @@ export const queryWebhookLogs = (
 
     if (externalService === 'all' || externalService === 'unmatched') {
         return requestGraphQL<WebhookLogsResult, WebhookLogsVariables>(
+            // V2ViaG9vazow is webhook ID 0.
```
Comment (ryanslade) [inline client/web/src/site-admin/webhooks/backend.ts:84]: Debugging comment?

Comment (ryanslade) [reply client/web/src/site-admin/webhooks/backend.ts:84]: Ah, nevermind, I get it.

Comment (pjlast) [reply client/web/src/site-admin/webhooks/backend.ts:84]: No, it's for future readers of the code wondering why `webhookID: "V2ViaG9vazow"` is part of the query

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.38 kb) | 0.00% (+0.38 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/pjlast/incoming-webhook-logs-no-webhook-id/compare-report.html;tab=live_object) for a full comparison between the commits 3e4fecd and feb1766 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by ryanslade — APPROVED** [review]
Feels a little hacky, but since we're going to have to rework this logic when we deprecate old style webhooks anyway I'm ok with it.

File: client/web/src/site-admin/webhooks/backend.ts [hunk 2]
```diff
@@ -81,9 +81,18 @@ export const queryWebhookLogs = (
...
+            // This is so that all webhook logs that have a webhookID
+            // associated with them are excluded.
```
Comment (mrnugget) [inline client/web/src/site-admin/webhooks/backend.ts:86]: Can't we add a `legacyOnly: true` flag for `webhookLogs`? We're going to remove it at some point any.

**Review by mrnugget — CHANGES_REQUESTED** [review]
I really think we shouldn't do that "0 means nothing" thing

**Comment by mrnugget** [comment]
> Feels a little hacky, but since we're going to have to rework this logic when we deprecate old style webhooks anyway I'm ok with it.

If it's possible to make it not hacky by spending 5min more in it, we should do that.

**Review by mrnugget — APPROVED** [review]

**Review by sashaostrikov — APPROVED** [review]
Nice, thanks!

**Pull request merged**